flink-tests/pom.xml

```diff
@@ -189,6 +189,22 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
```

Comment:
I notice you mentioned 

>Dependencies (does it add or upgrade a dependency): (yes / **no**)

in pull request description but it actually does upgrade a dependency. Could you please explain why this upgrade needed and correspondingly update the description?

Reply:
We have to add the hdfs related dependencies to use MiniDFSCluster in the tests. And the scope is only for test.

Reply:
>For the hdfs related dependencies, we have to add to use MiniDFSCluster in the tests. And the scope is only for test.

ok I see

> I think they stay in ZooKeeperHaServices is more reasonable.

make sense to me